lib/contentdm_translator.rb

```diff
@@ -121,33 +121,40 @@ def self.fst_field_for_collection(collection, license_key, contentdm_user_name,
       # no FTS
       error = "No full-text search (FTS) fields were configured on collection #{cdm_collection(at_id)}!"
     else
-    # good response
-      fts_field = doc.search("//field/type[text()='FTS']").first.parent.search('name').text
+      # good response
+      fts_field = doc.search("//field/type[text()='FTS']").first.parent.search('nickname').text
     end
 
     return error, fts_field
   end
 
   def self.export_work_to_cdm(work, username, password, license)
+    error, fieldname = fst_field_for_collection(work.collection, license, username, password)
```

Comment:
Should "fst_field" be "fts_field" (full text search)?

Reply:
I guess not -- maybe it was a typo the first time we wrote this?

Reply:
I'd like it updated to be fts_field instead of fst_field.  Better readability in the future.